pkg/scorecard.go

```diff
@@ -24,18 +24,12 @@ import (
 	"sync"
 
 	"github.com/google/go-github/v32/github"
-	"github.com/ossf/scorecard/checker"
+	"github.com/ossf/scorecard/lib"
 	"github.com/ossf/scorecard/roundtripper"
 	"github.com/shurcooL/githubv4"
 	"go.uber.org/zap"
 )
 
-type Result struct {
```

Comment:
Noticed this was changed from `Result` to `CheckResult`.

The `Result` implies what it is. 

What was the idea of having the `Check` prefix to the `Result`?

Reply:
The main reason is that we can have a single type which can be used across packages. So I have tried to delete as much new type definitions as possible across the code and have it defined in lib/ (now checker/).

Also, Result was simply CheckResult with check name (metadata, which is likely for the Repo, is never used). Now that checkName is part of CheckResult struct, I decided to remove it.